addons/payment_adyen/controllers/main.py

```diff
@@ -347,11 +356,6 @@ def _to_escaped_string(_value):
         # Calculate the HMAC with the binary representation of the signing string with SHA-256
         binary_hmac = hmac.new(binary_hmac_key, signing_string.encode('utf-8'), hashlib.sha256)
         # Calculate the signature by encoding the result with Base64
-        expected_signature = base64.b64encode(binary_hmac.digest())
-
-        # Compare signatures
-        if received_signature != to_text(expected_signature):
-            _logger.warning("ignored event with invalid signature")
-            return False
+        signature = base64.b64encode(binary_hmac.digest())
 
-        return True
+        return to_text(signature)
```

Comment:
do we still need these pycompat tools? it was for the python 2->3 transition

Reply:
Shouldn't we remove the tools then? If I find a tooling function that abstracts away the function I was going to use, I assume I must use the tooling as there could be many reasons for it to exist.

Reply:
After discussion, it turns out that this tool function can still be useful if we don't know the type of the source.